src/doc/rustc/src/platform-support/armv6k-nintendo-3ds.md

```diff
@@ -0,0 +1,104 @@
+# `armv6k-nintendo-3ds`
+
+**Tier: 3**
+
+The Nintendo 3DS platform, which has an ARMv6K processor, and its associated
+operating system (`horizon`).
+
+## Target maintainers
+
+- [@Meziu](https://github.com/Meziu)
+- [@AzureMarker](https://github.com/AzureMarker)
+- [@ian-h-chamberlain](https://github.com/ian-h-chamberlain)
+
+## Requirements
+
+This target is cross-compiled. `#![no_std]` crates should work without much
+additional effort.
+
+`std` is partially supported, but mostly works. Some APIs are unimplemented
+and will simply return an error, such as `std::process`. An allocator is provided
+by default.
+
+In order to support `std` APIs, binaries must be linked against the open-source
+[devkitARM toolchain](https://devkitpro.org/wiki/Getting_Started), notably
+using its `arm-none-eabi-gcc` as the linker.
+
+Additionally, some helper crates provide implementations of some `libc` functions
+use by `std`. These, or an alternate implementation of the relevant functions,
+are required to use `std`:
+
+- [`pthread-3ds`](https://github.com/Meziu/pthread-3ds) provides pthread APIs for `std::thread`.
+- [`linker-fix-3ds`](https://github.com/Meziu/rust-linker-fix-3ds) fulfills some other missing libc APIs.
```

Comment:
Maybe `ctru-sys`, if anything, though this falls in the considerations I had before…

Reply:
Since `ctru-sys` is a dependency of these crates, I think it's okay to omit here, since it's not required as a direct dependency, but I can add it if it seems relevant.

Reply:
True, I guess it should technically compile std with just those two (and dependencies).